app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md

```diff
@@ -98,7 +98,7 @@ Once you have Auth0 configured, you can configure the Dev Portal to use Auth0 fo
 
 7. Input the Client ID of the previously created admin application in Auth0 into the **Initial Client ID** field. input the Client Secret of the same Auth0 admin application into the Initial Client Secret field. **The Initial client secret will be stored in isolated, encrypted storage and will not be readable through any Konnect API.**
 
-8. Enter the **Client Audience** as the identifier value you set when configuring the API entity above.
+8. If you are using a custom domain for Auth0, enter the audience of the initial client as the **Client Audience** otherwise leave this field blank.
```

Comment:
```suggestion
8. If you are using a custom domain for Auth0, enter the audience of the initial client as the **Client Audience**. Otherwise, leave this field blank.
```